src/components/Users/Users.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import * as axios from "axios";
+
+import photo000 from "../../assets/images/photo000.png";
+
+import s from "./Users.module.css";
+
+class Users extends React.Component {
+  componentDidMount() {
+    axios
+      .get("https://social-network.samuraijs.com/api/1.0/users")
+      .then((response) => {
+        this.props.setUsers(response.data.items);
+      });
+  }
+
+  render() {
+    return (
+      <div>
+        {this.props.users.map((u) => (
```

Comment:
Remember - NEVER do `[...].map(...)` inside the "render" method of React Class component.
Consider creating 2 separate functions:
1. "getUsers" should hold the logic of mapping;
2. "getUser" should hold the logic of a map callback.

Reply:
done